### PR TITLE
Comm layout

### PR DESCRIPTION
**Pull request by ali-7**
control panel community layout -handlebars- #31 
view/add project  #10

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,6 @@
   "devDependencies": {
     "eslint": "^5.5.0",
     "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-plugin-hbs": "^0.1.1",
```
Comment (NouraldinS) [inline package.json:28]: Add this again, your teammates are using it.

**Review by NouraldinS — CHANGES_REQUESTED** [review]
Congrats @ali-7 Only one tiny chang requested, which you could probably argue. Good Job!

File: package.json (hunk 1 above)
Comment (ali-7) [reply package.json:28]: actually nobody use it , there's extinction to format the files in vscode we use it, and i asked before i remove it. :smile:

Comment (ali-7) [reply package.json:28]: and not all the modules trustworthy :P

Comment (NouraldinS) [reply package.json:28]: Yr call.

**Pull request merged**